modules/core/src/test/java/org/apache/ignite/internal/processors/cache/DataStorageConfigurationValidationTest.java

```diff
@@ -43,4 +43,28 @@ public void testWalSegmentSizeOveflow() throws Exception {
             }
         }, IllegalArgumentException.class, null);
     }
+
+    /**
+     * @throws Exception If failed.
+     */
+    public void testSetWalSegmentSizeShouldThrowExceptionWhenSizeLessThen512Kb() throws Exception {
+        final DataStorageConfiguration cfg = new DataStorageConfiguration();
+
+        GridTestUtils.assertThrows(null, new Callable<Void>() {
+            /** {@inheritDoc} */
+            @Override public Void call() throws Exception {
+                cfg.setWalSegmentSize(512 * 1024 - 1);
+                return null;
```

Comment:
According to Ignite code style, semantic units should be separated by empty lines.
See "Whitespaces and empty lines" section of https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines